test/common/network/cidr_range_test.cc

```diff
@@ -323,14 +323,14 @@ TEST(Ipv6CidrRange, InstanceConstSharedPtrAndLengthCtor) {
 
 TEST(Ipv6CidrRange, StringAndLengthCtor) {
   CidrRange rng;
-  rng = CidrRange::create("::ffff", 122); // Assignment operator.
+  rng = CidrRange::create("ff::ffff", 122); // Assignment operator.
```

Comment:
Sort of. [Wikipedia](https://en.wikipedia.org/wiki/IPv6_address#Default_address_selection) tells me an old version of the IPv6 spec considered `::/96` addresses (that is addresses with 96 bits of leading zeros) to be IPv6-mapped IPv4 addresses. It's been deprecated for a decade, but when OSX converts these addresses to a string they comes out as "::0.0.255.255" and the tests fail because they do string comparisons. Linux only provides the "mapped IPv4" style addresses with trailing dotted quads for the `::ffff:0:0/96` range (which replaced `::/96`). It seems to me that these test are concerned with how the low order bits of the address are manipulated and that using a different prefix doesn't change the meaning of the tests.

(I wrote a little program to round trip IP address strings to binary and back. https://gist.github.com/zuercher/0d0bc5a433fda423124c71afa2eb29e1)